test/scripts/imgtestlib.py

```diff
@@ -116,11 +116,17 @@ def s3_auth_args():
     return []
 
 
-def dl_s3_configs(destination):
+def dl_s3_configs(destination, distro=None, arch=None):
     """
     Downloads all the configs from the s3 bucket.
     """
     s3url = f"{S3_BUCKET}/{S3_PREFIX}/"
+    s3url = os.path.join(S3_BUCKET, S3_PREFIX)
```

Comment:
You're overwriting `s3url` here but also this is not the correct join operation for URIs which are different from filesystem paths, the previous one is correct (same goes for line 127).

Reply:
oops